README.md

```diff
@@ -6,6 +6,60 @@ Query API for the Tezos blockchain
 
 ## Running Conseil
 
+
+### Docker
+
+To build this project git clone it and run:
+
+```
+docker build -t conseil .
+```
+
+The Container can be configured using the following environment variables:
+
+Database Config:
+
+
+* DB_Host - default: db
+* DB_User - default: user
+* DB_Password - default: password
+* DB_Database: default: conseil
+
+
+
+Tezos Node Config:
+
+
+* XTZ_Scheme - http or https, default : http
+* XTZ_Host - default: node
+* XTZ_Prefix - prefix for api calls, default: ""
+* XTZ_Port - default 8732
+* XTZ_Network - default mainnet
```

Comment:
Currently only dev nodes are using containers, shouldn't the defaults be tailored towards dev environments which use alphanet?

Reply:
Most people using Conseil will use `mainnet`.